tests/test_e2e.py

```diff
@@ -505,6 +515,93 @@ def test_b2c_acquire_token_by_ropc(self):
             scope=["https://msidlabb2c.onmicrosoft.com/msidlabb2capi/read"],
             )
 
+    def test_arlington_acquire_token_by_ropc(self):
```

Comment:
Let's explore whether we could reuse most test logic among different clouds. These steps might work:

1. Change all the `def test_blah()` methods in this test class to `def _test_blah()` so that they won't run. This class would contain common test logic.
2. Create 2 sub-classes, such as `class WorldWideTestCase(LabBasedTestCase)` and `class ArlingtonTestCase(LabBasedTestCase)`, capture their difference in configuration. Define `def test_blah()` in these sub-classes to call their parent methods.

Reply:
I have separated this as per your suggestion. While doing that, I realized we can separate B2C and ADFS scenarios as well into their own subclasses as technically they are not using worldwide authorities. Thoughts ?

Reply:
(pls. see other comment) : pls. ensure you anticipate multiple clouds to be automated and that it should relatively easy to chose which ones to run, and e.g run all in a release build but only run one, two or a random selection for PR builds.

Reply:
To achieve Henrik's suggestion, let's rename the current test class from `SovereignCloudTestCase` to `ArlingtonCloudTestCase`, and define a property `environment = "azureusgovernment"`. All its test methods will reference such environment property. This way, in the future it would be very easy to expand to cover different clouds.

Reply:
Thanks for renaming this test class to `ArlingtonCloudTestCase`! Now we just need to remove all the "arlington" keyword from all its methods, so that in future we can easily copy-and-paste those test methods into other cloud, without needing to (remember to) change their names.